src/components/NumberInput.vue

```diff
@@ -1,21 +1,29 @@
 <template>
-  <div :class="className">
-    <label :for="id">
-      {{label}}<span v-if="isRequiredAsteriskShown" class="required-asterisk">*</span>
-    </label><br/>
+  <div
+    :class="className"
+  >
+    <label
+      :for="id"
+    >
+      {{ label }}<span
+        v-if="isRequiredAsteriskShown"
+        class="required-asterisk"
+      >*</span>
+    </label><br>
     <input
       :id="id"
+      ref="input"
       type="text"
       name="number"
       class="form-control"
       :data-cy="getCypressValue()"
-      :value="value"
+      :value="modelValue"
+      :style="inputStyle"
+      :maxlength="maxlength"
       @input="inputHandler($event)"
```

Comment:
Did you intentionally leave out the `.stop`?

Reply:
no, good catch added that in